android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -1,137 +1,117 @@
-import com.facebook.react.bridge.ReactContextBaseJavaModule;
-import com.facebook.react.bridge.ReactMethod;
-import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.WritableMap;
-import com.facebook.react.bridge.WritableNativeMap;
-import com.facebook.react.modules.core.DeviceEventManagerModule;
-import com.facebook.react.uimanager.ReactChoreographer;
 import com.facebook.react.uimanager.ViewManager;
 import com.facebook.soloader.SoLoader;
 
-import android.app.Activity;
 import android.content.Context;
-import android.content.Intent;
-import android.content.SharedPreferences;
 import android.content.pm.ApplicationInfo;
 import android.content.pm.PackageInfo;
 import android.content.pm.PackageManager;
-import android.os.AsyncTask;
-import android.provider.Settings;
-import android.view.Choreographer;
 
-import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.lang.reflect.Field;
-import java.lang.reflect.Method;
-
 import java.io.File;
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
 
 public class CodePush implements ReactPackage {
-    private static boolean needToReportRollback = false;
-    private static boolean isRunningBinaryVersion = false;
-    private static boolean testConfigurationFlag = false;
-
-    private boolean didUpdate = false;
-
-    private String assetsBundleFileName;
-
-    private static final String ASSETS_BUNDLE_PREFIX = "assets://";
-    private static final String BINARY_MODIFIED_TIME_KEY = "binaryModifiedTime";
-    private final String CODE_PUSH_PREFERENCES = "CodePush";
-    private static final String DEFAULT_JS_BUNDLE_NAME = "index.android.bundle";
-    private final String DOWNLOAD_PROGRESS_EVENT_NAME = "CodePushDownloadProgress";
-    private final String FAILED_UPDATES_KEY = "CODE_PUSH_FAILED_UPDATES";
-    private final String PACKAGE_HASH_KEY = "packageHash";
-    private final String PENDING_UPDATE_HASH_KEY = "hash";
-    private final String PENDING_UPDATE_IS_LOADING_KEY = "isLoading";
-    private final String PENDING_UPDATE_KEY = "CODE_PUSH_PENDING_UPDATE";
-    private final String RESOURCES_BUNDLE = "resources.arsc";
+
+    private static boolean mIsRunningBinaryVersion = false;
+    private static boolean mNeedToReportRollback = false;
+    private static boolean mTestConfigurationFlag = false;
+
+    private boolean mDidUpdate = false;
+
+    private String mAssetsBundleFileName;
 
     // Helper classes.
-    private CodePushNativeModule codePushNativeModule;
-    private CodePushPackage codePushPackage;
-    private CodePushTelemetryManager codePushTelemetryManager;
+    private CodePushNativeModule mCodePushNativeModule;
+    private CodePushUpdateManager mCodePushUpdateManager;
```

Comment:
Do we need the "CodePush" prefix in these? "mNativeModule", etc. might be sufficient in the context of the CodePush class.